docs/components/movement-sensor/gps/rtk-station.md

```diff
@@ -0,0 +1,157 @@
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": <sensor_name>,
+      "type": "movement_sensor",
+      "model": "rtk-station",
+      "attributes": {
+        "board": <board_name>,
+        "children": [
+          <list of children>
+        ],
+        "connection_type": <"serial" or "I2C">,
+        "i2c_attributes": {
+          "i2c_baud_rate": <>,
+          "i2c_addr": <>,
+          "i2c_bus": <>
+        },
+        "ntrip_attributes": {
+          "ntrip_addr": <>,
+          "ntrip_baud": <>,
+          "ntrip_password": <>,
+          "ntrip_path": <>,
+          "ntrip_username": <>
+        },
+        "serial_attributes": {
+          "serial_baud_rate": <>,
+          "serial_path": <>
+        },
+        "correction_source": <string>,
+        // non-NTRIP attributes:
+        "svin": <string>, //SurveyIn
+        "required_accuracy": <float64>,
+        "required_time_sec": <int>
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example: NTRIP" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "my-rtk-station",
+    "type": "movement_sensor",
+    "model": "rtk-station",
+    "children": [
+        "gps1"
+    ],
+    "attributes": {
+        "connection_type": "serial",
+        "ntrip_attributes": {
+            "ntrip_addr": "<NTRIP_address>",
+            "ntrip_baud": 38400,
+            "ntrip_password": "<password>",
+            "ntrip_path": "",
+            "ntrip_username": "<username>"
+        },
+        "correction_source": "ntrip"
+    }
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example: I2C" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "my-rtk-station",
+    "type": "movement_sensor",
+    "model": "rtk-station",
+    "children": [
+        "gps1"
+    ],
+    "attributes": {
+        "board": "board",
+        "connection_type": "serial",
+        "i2c_attributes": {
+            "i2c_baud_rate": 115200,
+            "i2c_addr": 111,
+            "i2c_bus": "<name_of_bus_on_board>",
+       },
+        "correction_source": "I2C",
+        "loc_accuracy": 10,
+        "svin": "time",
+        "time_accuracy": 60
+    }
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example: Serial/USB" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "my-rtk-station",
+    "type": "movement_sensor",
+    "model": "rtk-station",
+    "children": [
+        "gps1"
+    ],
+    "attributes": {
+        "connection_type": "serial",
+        "serial_attributes": {
+            "serial_baud_rate": 115200,
+            "serial_path": "/dev/serial/by-path/<device_ID>"
+        },
+        "correction_source": "serial"
+    }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Find NTRIP and configuration attribute information in the [GPS-RTK documentation](../gps-rtk/#attributes).

```

Comment:
This rtk station page is less complete than the others because it's especially experimental so I didn't think it warranted as much intention. loc accuracy, svin and time accuracy also aren't explained because I couldn't find that info, so I just left the example as-is and figure we can revisit.

Reply:
I wanted to include this in the refactor because it is currently in the docs, so this is an improvement over what's there currently, but this particular page still needs help (if it even remains as not a draft)